pkg/network/ovn_kubernetes_test.go

```diff
@@ -819,7 +826,50 @@ logfile-maxage=0`,
 			controlPlaneReplicaCount: 2,
 			disableMultiNet:          true,
 			enableMultiNetPolicies:   true,
-			enableAdminNetPolicies:   true,
+			enabledFeatureGates:      []configv1.FeatureGateName{configv1.FeatureGateAdminNetworkPolicy},
```

Comment:
`configv1.FeatureGateDNSNameResolver` is not set in the featuregates here

Reply:
I think this test is specific to ANP. The `enableAdminNetPolicies` field is changed to `enabledFeatureGates` for using the same field for different feature gates in different test cases. The test case for `DNSNameResolver` is added below.